packages/auth-passport/authService.js

```diff
@@ -5,10 +5,10 @@ var cloudUrl;
 var dialog;
```

Comment:
looks like you've replaced this variable with a class member, just need to remove it